src/Plugin/EntityBrowser/Widget/View.php

```diff
@@ -209,4 +210,41 @@ public function submit(array &$element, array &$form, FormStateInterface $form_s
     $this->selectEntities($entities, $form_state);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $options = [];
+    // Get only those enabled Views that have entity_browser displays.
+    $displays = Views::getApplicableViews('entity_browser_display');
+    foreach ($displays as $display) {
+      list($view_id, $display_id) = $display;
+      $view = $this->entityManager->getStorage('view')->load($view_id);
+      $options[$view_id . '.' . $display_id] = $this->t('@view : @display', array('@view' => $view->label(), '@display' => $view->get('display')[$display_id]['display_title']));
+    }
```

Comment:
There are many ways we could approach this problem. I'd say that we keep this simple and simply check if Views are enabled when displaying entity browser and display a warning. Additional to that we could do the same check/warning on the configuration UI page.

> And one more question: should this all be handled in this issue/PR, or should we just commit this and handle the dependencies in a different issue?

Separate issue/PR would work.